descriptors/ModuleDescriptor-template.json

```diff
@@ -118,6 +118,37 @@
         }
       ]
     },
+    {
+      "id": "invoice-storage.voucher-lines",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/voucher-storage/voucher-lines",
+          "permissionsRequired": ["invoice-storage.voucher-lines.collection.get"]
```

Comment:
Not an issue at all, just wanted to clarify maybe it makes sense to rename invoice-storage to vendor-storage in the permissionsRequired to match the endpoint name? In fact it's up to you. I just paid attention on this while implemented vaucher number generation because I have the same situation with permission and endpoint names.

Reply:
Agree with Slava, please rename permission to `voucher-storage.voucher-lines.collection.get`

Reply:
I named them according to the FOLIO naming conventions:https://dev.folio.org/guidelines/naming-conventions/
>The first portion is the exact name of the responsible module (back-end modules drop the mod- prefix).

I know this discussion has come up in the past that the naming conventions are too stringent, but I didn't see that there was any conclusion on that, so decided to stick with the rules

Reply:
Yeah that's my recollection too  - that there was no resolution...  the naming convention is indeed to strict and should probably be ammended to read something like:

> the first portion is the exact name of the responsible **interface**

It's really the last thing that prevents true portability of this interface (e.g. being able to move it to another module).   

I'm going to go out on a limb here and say we should break the rules here and make use permission names matching the interface, not the module name.

Reply:
ok, changed it to match the interface name